plugins/services/src/js/components/SDKPlans.tsx

```diff
@@ -63,6 +63,7 @@ const SDKPlans = componentFromStream(props$ => {
       return (
         <SDKPlansScreen
           key={index}
+          //@ts-ignore
```

Comment:
:not_sure_if:

Reply:
removed it and no errors seemed to pop up.